### ISSUE TITLE
Custom backup location

### ISSUE THREAD

**Issue by Deses** [jneilliii/OctoPrint-BackupScheduler#22]
**Is your feature request related to a problem? Please describe.**
The backups are stored on the same SD that the OctoPi instance is running on... if the SD dies the backups are gone too... and that's not a very good backup!

**Describe the solution you'd like**
I'd like to set a custom location to store backups, for instance a NAS or a USB connected device.

**Describe alternatives you've considered**
The only alternative right now is to move them manually.

**Comment by jneilliii** [comment]
Try one of the other plugins that work in conjunction with this one,  ie webdav or Google drive backup or onedrive backup, e.t.c. this is out of scope for what this plugin is intended to do, schedule backups.

**Comment by Deses** [comment]
Thank you for the response. I ended up learning about OctoPrint's events and setting up a mv command like you did in the wiki.

By the way, I saw that there are a bunch of closed issues (#15, #14, #13, #9, #5, #3) about cloud/NAS backups and to me it seems worth to update the README to include that this plugin won't add these features and that there are alternative plugins and events.

Hopefully that change will stop people from making new issues about the same topic.
